en/docs/getting-started/quick-start-guide/quick-start-guide-binary.md

```diff
@@ -107,48 +104,48 @@ Follow the steps below to expose the APIs via WSO2 API Microgateway.
 After the APIs are exposed via WSO2 API Microgateway, you can invoke an API with a valid token(JWT or opaque access token) or an API key.  Let's use WSO2 API Microgateway's API key endpoint to obtain an 
  API key in order to access the API.
    
-Below command will retrieve an APIKey token and set it to the shell variable TOKEN
+The below command will retrieve an APIKey token and set it to the shell variable `TOKEN`.
         
-``` java
+``` text
```

Comment:
better to use ```bash language for console commands so that it will provide better styling later if the markdown plugin improves. The language is already available